lib/sync/sync-processor.js

```diff
@@ -52,7 +52,8 @@ function recordProcessTime(startTime, success) {
 function markDatasetClientAsCompleted(datasetClientId, startTime, callback) {
   syncStorage.updateDatasetClient(datasetClientId, {syncLoopEnd: Date.now(), syncCompleted: true, syncScheduled: null}, function(err){
     if (err) {
-      syncUtil.doLog(datasetId, "error", "Error when update dataset client with id " + datasetClientId + ". error = " + util.inspect(err));
+      // TODO: datasetId here isn't defined, is it?
```

Comment:
@wei-lee syntax highlighting in my editor showed me that there's no `datasetId` and I think I agree. I'm not sure what the first param should be here, `syncUtil.SYNC_LOGGER` or `datasetClientId` or something?